tests/load-tests/ci-scripts/max-concurrency/collect-results.sh

```diff
@@ -78,6 +79,10 @@ ${csv_delim}ComponentsAvgTime\
 ${csv_delim}ComponentsMaxTime\
 ${csv_delim}PipelineRunAvgTime\
 ${csv_delim}PipelineRunMaxTime\
+${csv_delim}integrationTestsRunPipelineSucceededTimeAvg\
+${csv_delim}integrationTestsRunPipelineSucceededTimeMax\
+${csv_delim}deploymentSucceededTimeAvg\
+${csv_delim}deploymentSucceededTimeMax\
```

Comment:
```suggestion
${csv_delim}IntegrationTestsRunPipelineAvgTime\
${csv_delim}IntegrationTestsRunPipelineMaxTime\
${csv_delim}DeploymentSucceededAvgTime\
${csv_delim}DeploymentSucceededMaxTime\
```
Please keep the CSV hearder naming consistent